CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Draft
+- Date Field Modifier - February showing 30th and 31st [#2472](https://github.com/bigcommerce/cornerstone/pull/2472)
```

Comment:
@bc-vlad-dlogush fix the pr's number please

Reply:
Thanks!